contracts/libraries/PostLib.sol

```diff
@@ -331,15 +332,22 @@ library PostLib  {
             replyContainer.info.author,
             postContainer.info.communityId,
             UserLib.Action.EditItem,
-            UserLib.ActionRole.NONE,
+            isOfficialReply ? UserLib.ActionRole.CommunityModerator : 
+                UserLib.ActionRole.NONE,
             false
         );
         require(!CommonLib.isEmptyIpfs(ipfsHash), "Invalid_ipfsHash");
         require(userAddr == replyContainer.info.author, "You can not edit this Reply. It is not your.");
 
         if (replyContainer.info.ipfsDoc.hash != ipfsHash)
             replyContainer.info.ipfsDoc.hash = ipfsHash;
-        
+
+        if (isOfficialReply) {
+            postContainer.info.officialReply = replyId;
+        } else {
+            postContainer.info.officialReply = 0;
```

Comment:
When common user edits common reply, he unmarks official status for another reply
Will add unit test

Reply:
Fixed